src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkCorrectUsingMethodsInspection.kt

```diff
@@ -16,23 +23,25 @@ class SmkCorrectUsingMethodsInspection: SnakemakeInspection() {
             session: LocalInspectionToolSession
     ) = object : SnakemakeInspectionVisitor(holder, session) {
 
-        private fun checkAncient(node: PsiElement): PsiElement? {
-            if (node.text != "input")
-                return node
+        private fun checkAncient(psiEl: PsiElement): PsiElement? {
+            if (psiEl.text != SECTION_INPUT)
+                return psiEl
             return null
         }
 
-        private fun checkProtected(node: PsiElement): PsiElement? {
-            if (node.text != "output"
-                    && node.text != "log"
-                    && node.text != "benchmark")
-                return node
+        private fun checkProtected(psiEl: PsiElement): PsiElement? {
+            val psiElText = psiEl.text
+
+            if (psiElText != SECTION_OUTPUT
```

Comment:
Here it is good to have some generic impl, e.g. you could define a mapping from call name to allowed sections names and a generic method which works with mapping.

Reply:
Done